### ISSUE TITLE
Optional extra indentation of chained methods

### ISSUE THREAD

**Issue by qwemaze** [prettydiff/prettydiff#286]
Input:
``` javascript
new sql.Request()
.query(query.query)
.then((data) => {
	res.json({
		result: true
	});
})
.catch((e) => {
	e.status = e.status || 500;
	Log(e.stack || e);
	res.status(e.status).end();
});
```

Expected: 
``` javascript
new sql.Request()
.query(query.query)
.then((data) => {
	res.json({
		result: true
	});
})
.catch((e) => {
	e.status = e.status || 500;
	Log(e.stack || e);
	res.status(e.status)
    .end();
});
```

Actual:
``` javascript
new sql.Request()
	.query(query.query)
	.then((data) => {
		res.json({
			result: true
		});
	})
	.catch((e) => {
		e.status = e.status || 500;
		Log(e.stack || e);
		res.status(e.status)
			.end();
	});
```

**Comment by prettydiff** [comment]
Completed on my local.  I expect to release tonight.

**Comment by prettydiff** [comment]
Added new option to support this request named `nochainindent`.  Published in v1.16.29.
